recipe/meta.yaml

```diff
@@ -12,8 +12,9 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install --ignore-installed --no-deps .
+  skip: true  # [py<26 or py==30 or py==31 or py==32 or py==33]
 
 requirements:
   build:
```

Comment:
As we do not provide cross-compiling we can remove `python` from `req/build`

Reply:
The `python` requirement in the `build` section was removed

https://github.com/AnacondaRecipes/psutil-feedstock/blob/pbs_5.9.0_20211230T170103/recipe/meta.yaml#L20&&L21